yjit/src/core.rs

```diff
@@ -3817,7 +3816,7 @@ pub fn defer_compilation(
     };
 
     // Likely a stub since the context is marked as deferred().
-    let target0_address = branch.set_target(0, blockid, &next_ctx, ocb);
+    let target0_address = branch.set_target(0, blockid, &next_ctx, jit.get_outlined_cb());
```

Comment:
Maybe we could just make it `jit.get_ocb()` since it seems to get called somewhat often